src/Tickets/Commerce/Flag_Actions/Duplicated_Pending_Orders.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace TEC\Tickets\Commerce\Flag_Actions;
+
+use TEC\Tickets\Commerce\Order;
+use TEC\Tickets\Commerce\Status\Not_Completed;
+use TEC\Tickets\Commerce\Status\Pending;
+use TEC\Tickets\Commerce\Status\Status_Interface;
+use Tribe__Utils__Array as Arr;
+
+/**
+ * Class Duplicated_Pending_Orders, normally triggered when a given order is completed it will modify the status of all the
+ * other orders created with the same Hash cart key, which prevents leaving pending orders open when one was completed.
+ *
+ * @since    TBD
+ *
+ * @package  TEC\Tickets\Commerce\Flag_Actions
+ */
+class Duplicated_Pending_Orders extends Flag_Action_Abstract {
```

Comment:
I don't think this class name makes sense for what it is doing. Perhaps a better name would be more verb-like such a the following: `End_Duplicate_Pending_Orders`